packages/cli/src/scripts/octobase-service.ts

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env ts-node-esm
```

Comment:
It looks like this file is purely js. Maybe we can just use node instead and get rid of one extra dep?